mono/metadata/unity-utils.c

```diff
@@ -913,6 +913,34 @@ mono_unity_get_unitytls_interface()
 }
 
 // gc
+MONO_API void mono_unity_gc_set_mode(MonoGCMode mode)
+{
+#if HAVE_BDWGC_GC
+	switch (mode)
+	{
+		case MONO_GC_MODE_ENABLED:
+			if (GC_is_disabled())
+				GC_enable();
+			GC_set_disable_automatic_collection(FALSE);
+			break;
+
+		case MONO_GC_MODE_DISABLED:
+			if (!GC_is_disabled())
+				GC_disable();
+			break;
+
+		case MONO_GC_MODE_MANUAL:
+			if (GC_is_disabled())
+				GC_enable();
+			GC_set_disable_automatic_collection(TRUE);
+			break;
+	}
+#else
+	g_assert_not_reached ();
+#endif
+}
+
+// Deprecated. Remove when Unity has switched to mono_unity_gc_set_mode
```

Comment:
I'd prefer to not remove them too soon, as when we remove APIs I can't drop a new Mono into older unity (e.g. 2019.3).